Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -163,6 +183,7 @@ extension SwiftModule {
         buildSettings["PRODUCT_MODULE_NAME"] = c99name
         buildSettings["OTHER_SWIFT_FLAGS"] = "-DXcode"
         buildSettings["MACOSX_DEPLOYMENT_TARGET"] = "'10.10'"
+        buildSettings["LIBRARY_SEARCH_PATHS"] = "/usr/local/lib"
```

Comment:
K, understood. Yes please remove it, I worry we would forget about this sort of buried setting and then people will come to depend on it and then we can never remove it.